pyspades/world_c.cpp

```diff
@@ -506,6 +506,11 @@ inline void set_orientation_vectors(Vector *o, Vector *s, Vector *h)
 
 void reorient_player(PlayerType *p, Vector *orientation)
 {
+    Vector *o = orientation;
```

Comment:
I'd add a `.normalize()` and `.length()` method to `Vector` instead. Variable names are also not good. Also, I really don't like this code. `o` never gets used, but it is aliased to `orientation` and modified through that instead. This kind of trickery should be avoided to keep code understandable.